src/support/err.c

```diff
@@ -683,18 +685,41 @@ __wt_ext_strerror(WT_EXTENSION_API *wt_api, WT_SESSION *wt_session, int error)
  *     Progress message.
  */
 int
-__wt_progress(WT_SESSION_IMPL *session, const char *s, uint64_t v)
+__wt_progress(WT_SESSION_IMPL *session, WT_VERBOSE_CATEGORY category, const char *s, uint64_t v)
 {
     WT_DECL_RET;
     WT_EVENT_HANDLER *handler;
     WT_SESSION *wt_session;
+    size_t remain;
+    char operation[1024];
+    const char *msg;
+    bool is_json;
 
     wt_session = (WT_SESSION *)session;
+
     handler = session->event_handler;
+    is_json = FLD_ISSET(S2C(session)->json_output, WT_JSON_OUTPUT_PROGRESS);
+    msg = s == NULL ? session->name : s;
+    remain = sizeof(operation);
+
+    /* Generate message. */
+    WT_RET(__eventv_gen_msg(
+      session, operation, &remain, is_json, 0, NULL, 0, category, WT_VERBOSE_INFO, msg));
```

Comment:
Can probably update this `WT_VERBOSE_NOTICE`,  given WT-8397 is now merged :)

Reply:
Thanks for the update! Done in f104ec609999389a05019fb39dc76dfe0bb2c3a2.